provider/ansible.rb

```diff
@@ -235,66 +234,76 @@ def get_example(cfg_file)
       end
 
       def generate_resource(data)
-        # TODO: Azure switch
-        azure_generate_resource(data)
-
-        # target_folder = data.output_folder
-        # name = module_name(data.object)
-        # path = File.join(target_folder,
-        #                  "lib/ansible/modules/cloud/google/#{name}.py")
-        # data.generate(
-        #   data.object.template || 'templates/ansible/resource.erb',
-        #   path,
-        #   self
-        # )
+        # Multi Cloud Swith
+        if $cloud == 'MAC'
+          azure_generate_resource data
+        else
+          target_folder = data.output_folder
+          name = module_name(data.object)
+          path = File.join(target_folder,
+                           "lib/ansible/modules/cloud/google/#{name}.py")
+          data.generate(
+            data.object.template || 'templates/ansible/resource.erb',
+            path,
+            self
+          )
+        end
       end
 
       def generate_resource_tests(data)
-        # TODO: Azure switch
-        azure_generate_resource_tests(data)
-
-        # prod_name = data.object.name.underscore
-        # path = ["products/#{data.product.api_name}",
-        #         "examples/ansible/#{prod_name}.yaml"].join('/')
-
-        # return unless data.object.has_tests
-        # # Unlike other providers, all resources will not be built at once or
-        # # in close timing to each other (due to external PRs).
-        # # This means that examples might not be built out for every resource
-        # # in a GCP product.
-        # return unless File.file?(path)
-
-        # target_folder = data.output_folder
-
-        # name = module_name(data.object)
-        # path = File.join(target_folder,
-        #                  "test/integration/targets/#{name}/tasks/main.yml")
-        # data.generate(
-        #   'templates/ansible/integration_test.erb',
-        #   path,
-        #   self
-        # )
-
-        # # Generate 'defaults' file that contains variables.
-        # path = File.join(target_folder,
-        #                  "test/integration/targets/#{name}/defaults/main.yml")
-        # data.generate(
-        #   'templates/ansible/integration_test_variables.erb',
-        #   path,
-        #   self
-        # )
+        # Multi Cloud Swith
+        if $cloud == 'MAC'
```

Comment:
Similar comment as before. We might be able to minimize the changes to one line.

Reply:
Resolved.